blocks/offices/offices.js

```diff
@@ -49,11 +49,14 @@ function buildAgentCards(list, data) {
     cardsItem.className = 'cards-item';
     const cardImage = document.createElement('div');
     cardImage.className = 'card-image';
-    const tmpImage = 'https://main--hsf-commonmoves--hlxsites.hlx.page/media/images/no-profile-image.png';
-    const image = createOptimizedPicture(tmpImage, item.name, true);
-    image.style = 'padding-bottom: 75%';
-
-    cardImage.append(image);
+    const tmpImage = item.image ? item.image : 'https://main--hsf-commonmoves--hlxsites.hlx.page/media/images/no-profile-image.png';
```

Comment:
This needs to be dynamic or relative to root - the server can't change it to the prod domain or .live.